### DISCUSSION TITLE
Roll-back when leaving beta channel?

### DISCUSSION THREAD

**Discussion by soundflix** [sparkle-project/Sparkle#2657] [Q&A] [answered]
I there a way to offer users a stable, older version when they deselect the beta channel?
I am using `func allowedChannels(for updater: SPUUpdater) -> Set<String>`.

**Comment by zorgiepoo** [comment, marked as answer]
No, downgrades are not at all supported from a security perspective. In this case they’ll receive the next newer stable version. This is expected behavior and how Apple generally handles leaving a beta channel as well as far I’m aware. Downgrading software is also not always a supported migration path, depending on the software.
